src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -359,13 +359,13 @@ private function createChannelPricings(ProductVariantInterface $productVariant,
     private function createImages(ProductInterface $product, array $options): void
     {
         foreach ($options['images'] as $image) {
-            $imagePath = array_shift($image);
+            $imagePath = array_key_exists('path', $image) ? $image['path'] : array_shift($image);
```

Comment:
@pamil Should we actually `@trigger_error` here?

Reply:
We could add the deprecation in another PR targeting `master`, WDYT?

Reply:
Ah, that's right indeed!

Reply:
Done.